src/Entities/Api.js

```diff
@@ -19,8 +19,21 @@ export default class Api {
       this.refreshingToken
         .then(() => {
           this.tokenIsRefreshing = false;
+        })
+        .catch(() => {
+          this._logoutIfCan();
```

Comment:
then i catch na spokojnie by mogły być w jednej linii, a catch może być nawet krótszy! `.catch(this._logoutIfCan)`

Reply:
Poprawione, dzięki